app/models/book.rb

```diff
@@ -0,0 +1,8 @@
+class Book < ApplicationRecord
+  validates :genre, presence: true
+  validates :author, presence: true
+  validates :image, presence: true
+  validates :title, presence: true
+  validates :publisher, presence: true
+  validates :year, presence: true
```

Comment:
you could put everything in the same line :)
`validates :genre, :author, :image....., presence: true`